service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/RegistryManager.java

```diff
@@ -128,7 +237,6 @@ public void open()
      */
     public void close()
     {
-        // Codes_SRS_SERVICE_SDK_JAVA_REGISTRYMANAGER_34_087: [The function shall tell this object's executor service to shutdown]
```

Comment:
Why don't we like these comments?

Reply:
They clutter up the code base a lot

Reply:
Also because many of them are out of date. We haven't followed this convention of commenting in years.

Reply:
You think they offer no value to future readers?